src/routes/(app)/admin[[hash=admin_hash]]/reporting/+page.svelte

```diff
@@ -325,6 +326,13 @@
 						{#each order.payments as payment}
 							<tr class="hover:bg-gray-100 whitespace-nowrap">
 								<td class="border border-gray-300 px-4 py-2">{order.number}</td>
+								<td class="border border-gray-300 px-4 py-2"
+									>{payment.invoice?.number ??
+										`${order.number}-${
+											order.payments.findIndex((p) => p._id === payment._id) + 1
+										}`}</td
```

Comment:
it's more complicated that that, it should use:

```ts
t(
	'order.receipt.invoiceNumber',
	invoiceNumberVariables(data.order, payment)
)
```

And for partially paid orders, 

```ts
t(
	'order.receipt.proformaInvoiceNumber',
	invoiceNumberVariables(data.order, payment)
)
```